client/app/scripts/superdesk/filters.js

```diff
@@ -77,6 +77,11 @@ define([
                 }
             };
         })
+        .filter('truncateString', function() {
+            return function(inputString, limit, postfix) {
+                return _.trunc(inputString, {'length': limit, 'omission': postfix});
```

Comment:
We can supply a default value for 'omission' so we don't have to supply a value for it each time...